set.mm

```diff
@@ -614237,6 +614271,250 @@ number axioms (add ~ ax-10 , ~ ax-11 , ~ ax-13 , ~ ax-nul , and remove
+      ULUOUHULDEUIBNOUJUNCUMDEUNDEUJIISSPUMNUAUKTTBCUBVAURAUPDUSUPLUTUQCUSUPBKU
+      CUDUEUFUG $.
+
+    $( Closure for real subtraction.  Based on ~ subcl .  (Contributed by NM,
+       10-May-1999.)  (Revised by Steven Nguyen, 7-Jan-2023.) $)
+    rsubcl $p |- ( ( A e. RR /\ B e. RR ) -> ( A -R B ) e. RR ) $=
+      ( vx cr wcel wa crsub co cv caddc wceq crio rsubval rsubeu ancoms riotacl
+      wreu syl eqeltrd ) ADEZBDEZFZABGHBCIJHAKZCDLZDCABMUBUCCDQZUDDEUATUECBANOU
+      CCDPRS $.
+  $}
+
+  ${
+    $d x A $.  $d x B $.  $d x C $.
+    $( Relation between real subtraction and addition.  Based on ~ subadd .
+       (Contributed by NM, 20-Jan-1997.)  (Revised by Steven Nguyen,
+       7-Jan-2023.) $)
+    rsubadd $p |- ( ( A e. RR /\ B e. RR /\ C e. RR ) ->
+                   ( ( A -R B ) = C <-> ( B + C ) = A ) ) $=
+      ( vx cr wcel w3a crsub co wceq cv caddc crio wb wa rsubval eqeq1d 3adant3
+      wreu rsubeu oveq2 riota2 sylan2 3impb 3com13 bitr4d ) AEFZBEFZCEFZGABHIZC
+      JZBDKZLIZAJZDEMZCJZBCLIZAJZUGUHUKUPNUIUGUHOUJUOCDABPQRUIUHUGURUPNZUIUHUGU
+      SUHUGOUIUNDESUSDBATUNURDECULCJUMUQAULCBLUAQUBUCUDUEUF $.
+  $}
+
+  ${
+    rsubaddd.1 $e |- ( ph -> A e. RR ) $.
+    rsubaddd.2 $e |- ( ph -> B e. RR ) $.
+    rsubaddd.3 $e |- ( ph -> C e. RR ) $.
+    $( Relationship between subtraction and addition.  (Contributed by Mario
+       Carneiro, 27-May-2016.)  (Revised by Steven Nguyen, 8-Jan-2023.) $)
+    rsubaddd $p |- ( ph -> ( ( A -R B ) = C <-> ( B + C ) = A ) ) $=
+      ( cr wcel crsub co wceq caddc wb rsubadd syl3anc ) ABHICHIDHIBCJKDLCDMKBL
+      NEFGBCDOP $.
+  $}
+
+  ${
+    $d x y z $.
+    $( Real subtraction is an operation on the real numbers.  Based on ~ subf .
+       (Contributed by NM, 20-Jan-1997.)  (Revised by Steven Nguyen,
+       7-Jan-2023.) $)
+    rsubf $p |- -R : ( RR X. RR ) --> RR $=
+      ( vy vz vx cv caddc co wceq cr crio wcel wral cxp crsub wf rsubval rsubcl
+      wa eqeltrrd rgen2a df-rsub fmpt2 mpbi ) ADZBDEFCDZGBHIZHJZAHKCHKHHLHMNUFC
+      AHUDHJUCHJQUDUCMFUEHBUDUCOUDUCPRSCAHHUEHMCABTUAUB $.
+  $}
+
+  $( Addition and subtraction of equals.  (Contributed by Steven Nguyen,
+     8-Jan-2023.) $)
+  repncan2 $p |- ( ( A e. RR /\ B e. RR ) -> ( ( A + B ) -R A ) = B ) $=
+    ( cr wcel wa caddc co crsub wceq eqid readdcl simpl simpr rsubaddd mpbiri )
+    ACDZBCDZEZABFGZAHGBISSISJRSABABKPQLPQMNO $.
+
+  $( Addition and subtraction of equals.  Based on ~ pncan3 .  (Contributed by
+     Steven Nguyen, 8-Jan-2023.) $)
+  repncan3 $p |- ( ( A e. RR /\ B e. RR ) -> ( A + ( B -R A ) ) = B ) $=
+    ( cr wcel crsub co caddc wceq rsubcl w3a eqid rsubadd mpbii mpd3an3 ancoms
+    ) BCDZACDZABAEFZGFBHZPQRCDZSBAIPQTJRRHSRKBARLMNO $.
+
+  ${
+    mvrlladdd.1 $e |- ( ph -> A e. RR ) $.
+    mvrlladdd.2 $e |- ( ph -> B e. RR ) $.
+    mvrlladdd.3 $e |- ( ph -> ( A + B ) = C ) $.
+    $( Move real LHS left addition into RHS (real subtraction).  (Contributed
+       by Steven Nguyen, 8-Jan-2023.) $)
```

Comment:
The comment and the label are a little bit misleading. Maybe "Move LHS of a sum into RHS of a (real) difference."; label "reladdrsub".

Reply:
OK now